packages/odata-common/src/edm-types.ts

```diff
@@ -23,18 +24,17 @@ export type EdmTypeSameConverters =
   | 'Edm.Any'; // Represents currently unsupported EDM types like Edm.Geography.
 
 /**
- * @internal
+ * PLEASE REVIEW.
```

Comment:
[q] Why does this have to be exported?
```suggestion
 * Includes all EDM types that have different converters for OData version 2 and 4.
```

Reply:
Warning: EdmTypeDifferentConverters, defined at packages/odata-common/dist/edm-types.d.ts:13, is referenced by @sap-cloud-sdk/odata-common.EdmTypeCommon but not included in the documentation.

Reply:
And EdmTypeCommon is used in EdmTypeShared which is used in all packages.